nightfall-optimist/src/routes/proposer.mjs

```diff
@@ -397,14 +338,8 @@ router.post('/encode', async (req, res, next) => {
       )
       .encodeABI();
 
-    logger.debug({
```

Comment:
I see a lot of these debug messages (raw transaction) being removed, but they're quite useful sometimes. Are they being logged somewhere else? If not, I'd leave them here

Reply:
These logs in the routes are being carried out automatically when we set the `LOG_LEVEL=debug`. We log the payload and the response ;)